va/tlsalpn_test.go

```diff
@@ -26,6 +26,10 @@ import (
 	"github.com/letsencrypt/boulder/test"
 )
 
+func tlsalpnChallenge() core.Challenge {
```

Comment:
Is this being called anywhere?

Reply:
Good catch, I forgot to go back and update the rest of this file to match the others. Done.